git-extra/notepad

```diff
@@ -15,7 +15,12 @@ die "Usage: $0 <file>"
 
 if test -f "$1"
 then
-	unix2dos.exe "$1"
+	commitencoding="$(git config i18n.commitencoding)" ||
+		commitencoding=""
```

Comment:
Is this fallback needed? Can `git config i18n.commitencoding` return a value indicating failure? If it can, doesn't an output that matches `""|"utf-8"|"utf8")` still probably mean UTF-8 and an output that doesn't match probably mean something that's not UTF-8?